modules/metastorage/README.md

```diff
@@ -0,0 +1,46 @@
+# Metastorage
+
+The module for store and access to metadata. There is module is connected with this one:
+
+- metastorage-api - the module contains classes to access to metastorage service from another components.
```

Comment:
The module contains classes that other components can use to access the metastorage service.